kuiper-game/Domain/Ship/MiningScan.cs

```diff
@@ -27,17 +28,17 @@ public string StartEvent()
 
         public string EndEvent()
         {
-            if(DiceFactory.Roller().D100(chance,0))
-            {
-                var asteroid = CelestialBodies.Asteroid.Create("Scanned Asteroid",3.2, SolarSystemLocator.SolarSystem.SolarSystemService.GetBody("Sun"),CelestialBodies.CelestialBodyType.Asteroid );
-                var distance = SolarSystemLocator.SolarSystem.SolarSystemService.GetDistanceInAu(
-                    SolarSystemLocator.SolarSystem.Captain.Ship.CurrentLocation,
-                    asteroid
-                );
-                return $"Asteroid scanning complete. Found a candidate {distance}km from {SolarSystemLocator.SolarSystem.Captain.Ship.CurrentLocation}. Set a course to begin mining the asteroid.";
-            }
-            return $"Asteroid scanning complete, but found no asteroids with a yield within the scanners parameters.";
-
+            // if(DiceFactory.Roller().D100(chance,0))
+            // {
+            //     var asteroid = CelestialBodies.Asteroid.Create<Asteroid>("Scanned Asteroid",3.2, SolarSystemLocator.SolarSystem.SolarSystemService.GetBody("Sun"),CelestialBodies.CelestialBodyType.Asteroid );
+            //     var distance = SolarSystemLocator.SolarSystem.SolarSystemService.GetDistanceInAu(
+            //         SolarSystemLocator.SolarSystem.Captain.Ship.CurrentLocation,
+            //         asteroid
+            //     );
+            //     return $"Asteroid scanning complete. Found a candidate {distance}km from {SolarSystemLocator.SolarSystem.Captain.Ship.CurrentLocation}. Set a course to begin mining the asteroid.";
+            // }
+            // return $"Asteroid scanning complete, but found no asteroids with a yield within the scanners parameters.";
```

Comment:
I really hate commented code. 
It just offers so many opportunities for misunderstandings later down the line.
Eventually, someone will think "Was this supposed to be here or not?"

I think we should just delete it, if we're not using it.
If it's checked into the repo history, it can always be recovered, should we change our minds later on.